src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -556,6 +556,77 @@ Here are the explanation for the example template above:
+
+```
+-Dconductr.control-server.port = 9055
+```
+
+Restart the ConductR Core Service with the following command:
+
+```
+> sudo service conductr restart
+```
+
+HAProxy must then be configured to provide Basic Authentication and SSL termination for the Control Protocol. In this case, it will still be accessed via the default port `9005` externally. 
+
+Note: An SSL certificate will need to be generated and installed on the HAProxy host to enable two-way authentication for HTTPS. [OpenSSL](https://www.openssl.org/) can be used for this. While there are lots of resources available online for creating certificates, the following script can be used to generate a self-signed certificate for **testing purposes only**. You will need to substitute the fields of the `openssl req` comman with appropriate values for your server. It will create a folder called `ssl-conductr.cli` containing the certificates and a .pem file:
+
+```
+#!/bin/bash
+
+# Note - this script is for testing purposes only.
+
+set -e
+
+SCRIPT_DIR="$( cd "$(dirname $0)" && pwd)"
+CERT_NAME=ssl-"conductr.cli"
+SSL_CERT_DIR="$SCRIPT_DIR/$CERT_NAME"
+
+#rm $SSL_CERT_DIR/*
+mkdir -p $SSL_CERT_DIR
+
+echo "Generating SSL certs and keys into $SSL_CERT_DIR"
+openssl genrsa -out $SSL_CERT_DIR/$CERT_NAME.key 2048
+#This line creates the certificate - Most important fields are CN=.. and subjectAltName=... These are the domains that the cert can be used with. For the purposes of the ConductR CLI, this must cover the Cores at least.
+#For Dev servers, this would likely be (untested): *.ebiz.verizon.com
+openssl req -new -key $SSL_CERT_DIR/$CERT_NAME.key -sha256 -nodes -subj '/C=US/ST=California/L=San Francisco/O=Lightbend/OU=Prod Suite/CN=192.168.99.100/emailAddress=dev.null@lightbend.com' -out $SSL_CERT_DIR/$CERT_NAME.csr
+openssl x509 -req -days 365 -in $SSL_CERT_DIR/$CERT_NAME.csr -signkey $SSL_CERT_DIR/$CERT_NAME.key -out $SSL_CERT_DIR/$CERT_NAME.crt
+
+echo "Concat crt and key into pem file"
+cat $SSL_CERT_DIR/$CERT_NAME.crt $SSL_CERT_DIR/$CERT_NAME.key | tee $SSL_CERT_DIR/$CERT_NAME.pem
+```
+
+HAProxy is then configured with the path of our installed certificate and credentials to apply Basic Authentication to incoming ConductR Control API requests via port `9005`. To do this, we create a frontend `ctrl_frontend_secure` to handle incoming Control API requests, a userlist `ctrl_users` and a backend `ctrl_backend_auth` to direct to one of three available Conductr Cores, running on `172.17.0.1`,`172.17.0.2` and `172.17.0.3` listening to port `9055` as configured above.
```

Comment:
I agree, it is a run on sentence. I reformatted it into a list with a note about secure passwords that I think will be helpful.